Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1165,6 +1165,11 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+When the fixed cookie value is not used, a service affinity of a service with multiple
+ports is per service IP and port. Meaning that if a request from the same source
+to the same service but different ports is sent, each request might stick to
```

Comment:
```suggestion
to the same service but with different ports is sent, each request might stick to
```
Also: `each request might stick to different service endpoints` -> Can a request “stick”? Or should it be “connection” instead (or requests being “directed” to different endpoints)?

Reply:
Connection! Thanks.